scripting/include/srccoop/classdef.inc

```diff
@@ -1850,31 +2398,80 @@ methodmap IServerGameDLL
 
 methodmap CTraceRay < Handle
 {
-	public CTraceRay(const float vecPosition[3], const float vecRayType[3], const int iFlags, const RayType rayType, const TraceEntityFilter traceFilter, const any pData)
+	public CTraceRay(const float vecPosition[3], const float vecRayType[3], const int iTraceMask, const RayType eRayType, const TraceEntityFilter fnFilter, const any pData)
 	{
-		return view_as<CTraceRay>(TR_TraceRayFilterEx(vecPosition, vecRayType, iFlags, rayType, traceFilter, pData));
+		return view_as<CTraceRay>(TR_TraceRayFilterEx(vecPosition, vecRayType, iTraceMask, eRayType, fnFilter, pData));
+	}
+
+	public Handle GetHandle()
```

Comment:
why is this needed?

Reply:
Are you talking about `GetHandle`? I'm using it like this to limit the number of casts.

If you are talking about the other member functions, I need them for the implementation of FVisible.

Reply:
But if you inherit from Handle you should not need to cast anyway.

Reply:
Oversight on my part. Fixed this by removing `GetHandle` and using `this` directly.